src/services/postService.js

```diff
@@ -100,4 +136,19 @@ module.exports = {
         }
       });
   },
+  extractFromSteemitResponse: function(posts) {
+    return posts.map(post => (
+      { permLink: post.permlink, name: post.author }
+    ))
+  },
+  findByPermLinkAndAuthor: function(permLink, name) {
+    return PostModel.findOne({
+      where: { permLink },
+      include: [
+        { model: UserModel,
+          where: { name }
+        }
+      ]
```

Comment:
this looks weird. It's not a big deal but could we make it
```
...
include: [
   {
      model: UserModel,
      where: { name }
   }
]
...
```

Reply:
sure